python/tests/special_kz.py

```diff
@@ -80,20 +80,28 @@ def test_special_kz(self):
         theta = math.radians(23)
 
         start = time()
-        Rmeep_no_kz = self.refl_planar(theta, False)
-        t_no_kz = time() - start
+        Rmeep_complex = self.refl_planar(theta, 'complex')
+        t_complex = time() - start
 
         start = time()
-        Rmeep_kz = self.refl_planar(theta, True)
-        t_kz = time() - start
+        Rmeep_real_imag = self.refl_planar(theta, 'real/imag')
+        t_real_imag = time() - start
+
+        start = time()
+        Rmeep_3d = self.refl_planar(theta, '3d')
```

Comment:
The 3d simulation can probably be removed (in order to save time) since this particular test is only for the `special_kz=True` feature.